src/django/pfb_analysis/views.py

```diff
@@ -12,24 +12,26 @@
 from rest_framework.viewsets import ModelViewSet
 from rest_framework.response import Response
 
-from pfb_analysis.models import AnalysisJob, Neighborhood
-from pfb_analysis.serializers import AnalysisJobSerializer, NeighborhoodSerializer
 from pfb_network_connectivity.pagination import OptionalLimitOffsetPagination
-from pfb_network_connectivity.filters import (OrgAutoFilterBackend,
-                                              SelfUserAutoFilterBackend,
-                                              AnalysisJobStatusFilterSet)
+from pfb_network_connectivity.filters import OrgAutoFilterBackend
 from pfb_network_connectivity.permissions import IsAdminOrgAndAdminCreateEditOnly, RestrictedCreate
 
+from .models import AnalysisJob, Neighborhood
+from .serializers import AnalysisJobSerializer, NeighborhoodSerializer
+from .filters import AnalysisJobFilterSet
+from .functions import ObjectAtPath
+
 
 class AnalysisJobViewSet(ModelViewSet):
     """For listing or retrieving analysis jobs."""
 
-    queryset = AnalysisJob.objects.all()
+    queryset = AnalysisJob.objects.all().annotate(
+        overall_score=ObjectAtPath('overall_scores', ('overall_score', 'score_normalized')))
```

Comment:
Yeah, I agree, but https://docs.djangoproject.com/en/1.10/ref/contrib/postgres/fields/#jsonfield is focused on filtering and I don't see anything that would fit the bill at https://docs.djangoproject.com/en/1.10/ref/models/expressions/

The double-underscore syntax has been extended to traversing JSONField values in some contexts, e.g.
```
AnalysisJob.objects.filter(overall_scores__overall_score__score_normalized__gt=1)
```
works, but
```
AnalysisJob.objects.annotate(overall_score=F('overall_scores__overall_score__score_normalized'))
```
says "FieldError: Cannot resolve keyword 'overall_score' into field. Join on 'overall_scores' not permitted."